executor/executor_test.go

```
@@ -5887,6 +5887,12 @@ func (s *testClusterTableSuite) TearDownSuite(c *C) {
 	s.testSuiteWithCliBase.TearDownSuite(c)
 }
 
+func (s *testSuiteP1) TestPrepareLoadData(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec(`prepare stmt from "load data local infile '/tmp/load_data_test.csv' into table test";`)
```

It should return an error. You could use `tk.ExecToErr`